packages/coinjoin/src/backend/CoinjoinBackendClient.ts

```diff
@@ -129,8 +127,7 @@ export class CoinjoinBackendClient {
     }
 
     fetchMempoolTxids(options?: RequestOptions): Promise<string[]> {
-        return this.scheduleGet(
-            this.wabisabi,
+        return this.fetchFromWabisabi(
```

Comment:
Just for my curiousity, is this providing all some sort of filtered mempool txs? Why using wabisabi instead of blockbook WS?

Reply:
Blockbook doesn't have any means how to get all existing mempool transactions, so we fetch mempool txid list from wabisabi and then the transactions one-by-one from blockbook. Will be somewhat improved by subscribing to newly added mempool txs as a followup.